src/runtime_src/core/edge/drm/zocl/zocl_aie.c

```diff
@@ -272,8 +273,13 @@ zocl_aie_getcmd_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
 	mutex_lock(&aie->aie_lock);
 	while (list_empty(&aie->aie_cmd_list)) {
 		mutex_unlock(&aie->aie_lock);
-		if (wait_event_interruptible(aie->aie_wait_queue,
-		    !list_empty(&aie->aie_cmd_list))) {
+		/* 
+ 		 * Return greater then 0 if condition true before timeout,
+ 		 * 0 when time out, else -ERESTARTSYS.
+ 		 */
+		ret = wait_event_interruptible_timeout (aie->aie_wait_queue,
```

Comment:
Please remove the blank space between function name and '('

Reply:
fixed